frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -432,11 +439,12 @@ export type ListPageFilterProps<D = any> = {
   onFilterChange: OnFilterChange;
   hideColumnManagement?: boolean;
   nameFilter?: string;
+  rowSearchFilters?: RowSearchFilter<D>[];
 };
 
 export type UseListPageFilter = <D, R>(
   data: D[],
-  rowFilters?: RowFilter<R>[],
+  rowFilters?: (RowFilter<R> | RowSearchFilter<R>)[],
```

Comment:
Instead of repeating `RowFilter | RowSearchFilter` or `RowFilter<R> | RowSearchFilter<R>` type unions across the code, I'd suggest introducing some type alias, for example:

```ts
export type AnyRowFilter<R = any> = RowFilter<R> | RowSearchFilter<R>;
```

placed right after the `RowSearchFilter` type definition.